utils/parseTokenId.ts

```diff
@@ -0,0 +1,49 @@
+import { LSP8_TOKEN_ID_FORMAT } from '@lukso/lsp-smart-contracts'
+import { hexToNumber, hexToUtf8 } from 'web3-utils'
+
+/**
+ * Parse token ID based on the format
+ *
+ * @param tokenId
+ * @param tokenIdFormat
+ * @returns
+ */
+export const parseTokenId = (tokenId?: string, tokenIdFormat?: number) => {
```

Comment:
Can't use like that since this is an const object, but we can create new type from it like this:

```
export type TokenIdFormatKey = keyof typeof LSP8_TOKEN_ID_FORMAT
export type TokenIdFormatValue = (typeof LSP8_TOKEN_ID_FORMAT)[TokenIdFormatKey]
```

Reply:
ok